authors/apps/authentication/utils.py

```diff
@@ -19,3 +29,72 @@ def validate_image(avatar_name):
         APIException.status_code = status.HTTP_400_BAD_REQUEST
         raise APIException({"image":
                             "Only '{}', '{}', '{}' files are accepted".format(*ok_formats)})  # noqa
+
+
+def send_password_reset_link(to_email, username, token, callback_url):
+    domain = settings.DOMAIN
+    html_content = render_to_string('password_reset.html',
+                                    {'username': username,
+                                     'callback_url': callback_url,
+                                     'token': token, 'domain': domain})
+    subject = 'Password Reset'
+    from_email = settings.EMAIL_HOST_USER
+    msg = EmailMessage(subject, html_content, from_email, [to_email])
+    msg.content_subtype = 'html'
+    msg.send()
+    return Response({"message":
+                    "A password reset email has been"
+                     " sent to your email account. "
+                     "If you don't see the email,"
+                     " check other places it might be, "
+                     "like your junk, spam,"
+                     " social, or other folders."},
+                    status=status.HTTP_200_OK)
+
+
+class TokenHandler:
```

Comment:
Please use the function in the `core` app under `utils` ... This feature is already implemented there

Reply:
I will make changes.